gsy_framework/live_events/b2b.py

```diff
@@ -0,0 +1,84 @@
+from datetime import datetime
+from typing import Dict, Callable
+
+from gsy_framework.enums import AvailableMarketTypes
+from gsy_framework.utils import str_to_pendulum_datetime
+
+
+class B2BLiveEvents:
+    """Live events that support b2b trading."""
+    ENABLE_TRADING_EVENT_NAME = "enable_trading"
+    DISABLE_TRADING_EVENT_NAME = "disable_trading"
+    POST_ORDER_EVENT_NAME = "post_order"
+    REMOVE_ORDER_EVENT_NAME = "remove_order"
+
+    @classmethod
+    def is_supported_event(cls, event_name: str) -> bool:
+        """Check if the event name is one of the supported live events."""
+        return event_name in [
+            cls.ENABLE_TRADING_EVENT_NAME, cls.DISABLE_TRADING_EVENT_NAME,
+            cls.POST_ORDER_EVENT_NAME, cls.REMOVE_ORDER_EVENT_NAME
+        ]
+
+
+class LiveEventArgsValidator:
+    """Validator class for the forward strategy live event arguments."""
+    def __init__(self, logging_function: Callable):
+        self._logger = logging_function
+
+    def validate_start_trading_event_args(self, args: Dict) -> bool:
```

Comment:
I am not sure what our consensus was when we discussed about using `validate` in method names, but I would expect here that the method would raise an exception and not return anything. Do you agree?
If so, I suggest to rename to `are_start_trading_event_args_valid`.

Reply:
I agree, and I will rename all the methods in this class to reflect this. Thanks!